### PR TITLE
Updating Logic to add a timer for every second

### PR DESCRIPTION
**Pull request by Dloranger**
New timer (every 1 second) to support new site status module

### CODE REVIEW

**Comment by sm0svx** [comment]
Wow! I seem to have some work to do this weekend. Looking at this PR now.

File: src/svxlink/svxlink/Logic.cpp [hunk 1]
```diff
@@ -1484,6 +1489,22 @@ void Logic::everyMinute(AtTimer *t)
   timeoutNextMinute();
 } /* Logic::everyMinute */
 
+void Logic::timeoutNextSecond(void)
+{
+  struct timeval tv;
+  gettimeofday(&tv, NULL);
+  struct tm *tm = localtime(&tv.tv_sec);
+  tm->tm_min += 0;
```
Comment (sm0svx) [inline src/svxlink/svxlink/Logic.cpp]: Looks a bit strange with "+= 0" since it's a noop but I'll fix that.

**Comment by sm0svx** [comment]
Logic.tcl, RepeaterLoigic.tcl, SimplexLogic.tcl also need to be changed to include the every_second event handler function. Fixed that. Also renamed the addTimerTickSubscriber function to addMinuteTickSubscriber. New function addSecondTickSubscriber.

**Comment by dl7ata** [comment]
In the current distribution, I am missing  the procedure
                                                                                                                                 
#Executed once every second
proc every_second {} {
  Logic::every_second;                                                                                                              
}

in /usr/share/svxlink/events.d/RemoteLogic.tcl.
Otherwise I got an error in the log: 

 *** ERROR: Unable to handle event: RemoteLogic::every_second in logic RemoteLogic (invalid command name "RemoteLogic::every_second")

**Comment by Dloranger** [comment]
I don't see in the source of RemoteLogic.Tcl in the matster branch so I can't verify directly but I suspect you need to add the following to the RemoteLogic.tcl file

#
# Executed once every second
#
proc every_second {} {
  Logic::every_second;
}

**Comment by dl7ata** [comment]
I think this might be the problem. An related RemotLogic.tcl with new syntax and params is missing in the latest branch.

**Comment by sm0svx** [comment]
There is no RemoteLogic in standard SvxLink so that is a local addition or code from a forked repository.

**Comment by dl7ata** [comment]
Ahhh - I can't remember when installing the remotelogic.
Thanks Tobias! 
I will consider when updating my other stations.

**Pull request merged**